test/ghost_derivative_operators_interface.jl

```diff
@@ -7,7 +7,7 @@ using DiffEqOperators, Test
     A = L * Q
 
     @test ndims(A) == 2
-    @test size(A) == (10, 10)
-    @test size(A, 1) == 10
-    @test length(A) == 100
+    @test_broken size(A) == (10, 10)
```

Comment:
how did these break?

Reply:
one of my "rebase" commits changes `A.L.len` to `size(A.L, 2)` in the definition of `size(::GhostDerivativeOperator)` which all the broken tests rely on.  I don't see any reason whatsoever why I would have done that -- both master and xtalax have `A.L.len` and I'm aware that those aren't the same thing. So I can fix that, but where should I commit?

Reply:
PR to this branch.

Reply:
AH, my mistake.  Disregard my last comment -- I was, today, looking at xtalax:master rather than xtalax:#168 to figure out what I had been rebasing towards.  Luckily for my own personal sanity, I wasn't stupid enough to introduce my own ideas when I was rebasing.  Unfortunately, though, I'm not sure I can fix this broken test.  I marked it as broken because I saw that I could not fix it and I wanted to get past it to any tests I might be able to fix.  xtalax:#168 does use `size(A.L, 2)` in its definition of `size(::GhostDerivativeOperator)`.  That means that `size(A)` will return `(size(A.L, 2), size(A.L, 2))` which will return `(12,12)` in a very straightforward manner because in the constructor for CenteredDifference it sets the second dimension's size to be `10+2`.  I don't know which definition of `size` (or maybe the constructor?) is wrong.  I can start trying to fix the code's semantics, but that'll be a longer process.

Reply:
I would revert that change I think?

Reply:
Reverted and passing.  @ChrisRackauckas how often do you want me to PR from my personal branch into this branch?